lib/admin_form_builder.rb

```diff
@@ -0,0 +1,20 @@
+class AdminFormBuilder < GenericFormBuilder
```

Comment:
I'm expecting us to rename this anyway when we extract it to a gem.  Something with `govuk` in it?  Open to all suggestions.

The point of admin is that it defaults to classes and layouts appropriate for admin applications.